internal/scaffold/ansible/molecule_test_local_prepare.go

```diff
@@ -67,4 +68,4 @@ const moleculeTestLocalPrepareAnsibleTmpl = `---
       - role.yaml
       - role_binding.yaml
       - service_account.yaml
-`
+` //nolint:golint
```

Comment:
Why added `//nolint:golint` ? Shoul no be `lll`?

Reply:
Or nothing. @I-am-Dishon

Reply:
i did not see the change before committing but it has no significance where it is i will make commit to remove the it

Reply:
Solved tks.